docs/networking.md

```diff
@@ -29,7 +29,8 @@ This section uses three types of server-side software objects to represent netwo
 * A network, which is a virtual Ethernet switch on a host. Network objects have a name and description, a UUID, and the collection of VIFs and PIFs connected to them.
 
 `xe` CLI, Xen Orchestra or XCP-ng center allow you to configure networking options. You can control the NIC used for management operations, and create advanced networking features such as VLANs and NIC bonds.
-Networks
+
+### Networks
```

Comment:
I think it should be only `##` as for VLANs, Bonds etc
@stormi what do you think?

Reply:
I see, might be, I'll update if we agree on that